pkg/ebpf/net_events.go

```diff
@@ -233,23 +233,27 @@ func (t *Tracee) processNetEvents(ctx gocontext.Context) {
 			// continue without checking for error, as packetContext will be valid anyway
 			packetContext, networkThread, _ := t.getPcapContextFromTid(netEventMetadata.HostTid)
 
-			if netEventMetadata.NetEventId == NetPacket {
+			if isNetEvent(netEventMetadata.NetEventId) {
 				var netCaptureData bufferdecoder.NetCaptureData
 				err = netDecoder.DecodeNetCaptureData(&netCaptureData)
 				if err != nil {
 					t.handleError(err)
 					continue
 				}
+				PacketStartOff := netEventMetadata.GetSizeBytes() + netCaptureData.GetSizeBytes() // packet payload offset is NetEventMetadata + NetCaptureData
```

Comment:
The comment here is the same as the code. It has no added value.
Please write in the comment why that packet start after both of this structs (maybe put it above the line).